msteams-platform/concepts/build-and-test/deep-link-workflow.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Deep link to a workflow in Teams
+author: v-npaladugu
+description: Learn how to create deep links to a specific task in Microsoft Teams and navigate using them in your Teams.
+ms.topic: conceptual
+ms.author: surbhigupta
+ms.localizationpriority: high
+---
+
+# Deep link to a workflow in Teams
+
+You can create a deep link to perform a specific task in Teams, such as to create a new chat, open a scheduling dialog, and navigate to audio-video call.
+
+## Deep link to start a new chat
+
+Applications can start a new chat with a list of users and provide additional information such as chat name and draft message by using the following format:
+
+`https://teams.microsoft.com/l/chat/0/0?users=<user1>,<user2>,...&topicName=<chat name>&message=<precanned text>`
+
+Example: `https://teams.microsoft.com/l/chat/0/0?users=joe@contoso.com,bob@contoso.com&topicName=Prep%20For%20Meeting%20Tomorrow&message=Hi%20folks%2C%20kicking%20off%20a%20chat%20about%20our%20meeting%20tomorrow`
+
+The query parameters are:
+
+* `users`: The comma-separated list of user IDs representing the participants of the chat. The user that performs the action is always included as a participant. Currently, the User ID field supports the Microsoft Azure Active Directory (Azure AD) UserPrincipalName, such as an email address only.
+* `topicName`: An optional field for chat's display name if a chat has three or more users. If this field isn't specified, the chat's display name is based on the names of the participants.
+* `message`: An optional field for the message text that you want to insert into the current user's compose box while the chat is in a draft state.
+
+To use this deep link with your bot, specify this as the URL target in your card's button or tap action through the `openUrl` action type. Apps can also use Teams Java Script SDK 2.0 to create this without having to manually prepare the deep link. Following is an example using Teams JS SDK:
+
+```javascript
+if(chat.isSupported()) {
+    const chatPromise = chat.openGroupChat({ users: ["joe@contoso.com","bob@contoso.com"], topic: "Prep For Meeting Tomorrow", message: "Hi folks kicking off chat about our meeting tomorrow"});
+    chatPromise.
+      then((result) => {/*Successful operation*/}).
+      catch((error) => {/*Unsuccessful operation*/});
+}
```

Comment:
Add a note that if a chat already exists, it will open that.

Reply:
Added the note